lua/copilot/client.lua

```diff
@@ -19,7 +19,7 @@ end
 
 M.merge_server_opts = function (params)
   return vim.tbl_deep_extend("force", {
-    cmd = { "node", require("copilot.util").get_copilot_path(params.plugin_manager_path) },
+    cmd = { "node",  debug.getinfo(1).source:sub(2):gsub("lua/copilot/client.lua", "") .. "copilot/index.js"},
```

Comment:
change this to be:
`cmd = { "node", require("copilot.util").get_copilot_path() },`